unity-environment/ProjectSettings/EditorBuildSettings.asset

```diff
@@ -4,4 +4,5 @@
 EditorBuildSettings:
   m_ObjectHideFlags: 0
   serializedVersion: 2
+  m_Scenes: []
```

Comment:
Can we add this file to the gitignore maybe ?

Reply:
We don't want to do that in case it changes between Unity versions, and we accidentally break something.